### PR TITLE
Brooke-harvey-blog-international

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/aYTK18Dc/3066-international-teacher-case-study-interview

### CODE REVIEW

**Comment by EllieNodder** [comment]
@peteryates I tried to add a thumbnail image but it didn't seem to like it and said it couldn't find it when I put it into the frontmatter. It has also chopped off the top of her head 😣 could you help pls? 😅

**Comment by peteryates** [comment]
Yep will take a look this afternoon

**Comment by EllieNodder** [comment]
@peteryates the picture on the blog page is chopping the top of her head off, can we fix it?

**Comment by peteryates** [comment]
@EllieNodder - done!

The image will be placed in the centre of the box so things near the edge of the frame might get chopped. All I did to correct it here was to [re-crop so her face is more central](https://github.com/DFE-Digital/get-into-teaching-app/pull/2517/commits/9f00fbf835a409c04876b1e0bcdd2db3d17a5496).

**Comment by EllieNodder** [comment]
@gemmadallmandfe @valentine-carter Brooke has got back saying she's happy with the blog - as is Laura - so I'm happy for this to be merged if you two are

File: app/views/content/blog/my-move-from-canada-to-teach-in-england.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+My advice would be that:
+
+- moving across the world takes a certain type of person, so it’s good to be very clear with yourself about why you’re making the move
+ 
+- you should be in direct contact with the school you’re applying to and form a dialogue with someone there
+ 
+- you can love your school and love your class, but leadership shapes everything — make sure you respect and trust the people who employ you
+ 
+- you'll need to put a network in place for when you're no loner running on pure adrenaline — there's a bit of a lull and you start to want help and support
```
Comment (gemmadallmandfe) [inline app/views/content/blog/my-move-from-canada-to-teach-in-england.md:69]:
```suggestion
- you'll need to put a network in place for when you're no longer running on pure adrenaline — there's a bit of a lull and you start to want help and support
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**